poprox-db/migrations/versions/2025_07_01_0957-d1f9922c638b_create_top_story_packages_table.py

```diff
@@ -0,0 +1,94 @@
+"""Create top story packages table
+
+Revision ID: d1f9922c638b
+Revises: 72901f4ab61c
+Create Date: 2025-07-01 09:57:31.375838
+
+"""
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = 'd1f9922c638b'
+down_revision: Union[str, None] = '72901f4ab61c'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # Create the new table and link it to the entities table
+    op.create_table(
+        "top_story_packages",
+        sa.Column("package_id", sa.UUID, primary_key=True, nullable=False),
+        sa.Column("entity_id", sa.UUID, nullable=True),
+        sa.Column("as_of", sa.DateTime, nullable=False),
+        sa.Column("created_at", sa.DateTime, nullable=False, server_default=sa.text("NOW()")),
```

Comment:
any particular reason to have `as_of` and `created_at` -- Like I think I can see why they would be different values in general, but having both feels a bit redundant.

Reply:
Makes sense to me. :shipit: